libs/designer-ui/src/lib/styles.less

```diff
@@ -40,6 +40,9 @@
 @import "./floatingactionmenu/_floatingactionmenu.less";
 @import "./monitoring/monitoring.less";
 @import "./monitoring/statuspill/statuspill.less";
+@import "./templates/templatesPanelContent.less";
+@import "./templates/templatesPanelFooter.less";
+@import "./templates/templatesPanelHeader.less";
 @import "./templates/templatesParametersField.less";
```

Comment:
Lets try to keep less styling files based on their component, like panel, header and footer in one, and tabs can have separate.